src/app/modules/form/form.errors.ts

```diff
@@ -26,7 +26,7 @@ export class PrivateFormError extends ApplicationError {
    * @param formTitle Extra meta for form title
    */
   constructor(
-    message = 'If you think this is a mistake, please contact the agency that gave you the form link.',
+    message = 'features.publicForm.errors.private',
```

Comment:
i think we should not put such strings into the server. We can map the server error messages to a frontend mapping. Change this to a (shared) constant instead then perform frontend mapping to the translated strings?

Reply:
Agree, on not putting the translation keys into the server.

I'm rather against server providing message to the FE to display them. We should ideally be passing error codes (hoping to get to this soon) instead, and keeping additional messages to be in English for those that are using calling our APIs programmatically. Adding this will likely be introducing patterns that will go against what we want to do in the future (BE passes FE by codes, and FE does the actual translation mapping).

Reply:
ok, will drop this change now that I've drawn attention to this part of the codebase and wait for @KenLSM or someone else to refactor error mechanism.